homeassistant/components/motionblinds_ble/config_flow.py

```diff
@@ -0,0 +1,210 @@
+"""Config flow for MotionBlinds BLE integration."""
+from __future__ import annotations
+
+import logging
+import re
+from typing import Any
+
+from bleak.backends.device import BLEDevice
+import voluptuous as vol
+
+from homeassistant.components import bluetooth
+from homeassistant.components.bluetooth import BluetoothServiceInfoBleak
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.selector import (
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+)
+
+from .const import (
+    CONF_ADDRESS,
+    CONF_BLIND_TYPE,
+    CONF_LOCAL_NAME,
+    CONF_MAC_CODE,
+    DOMAIN,
+    ERROR_COULD_NOT_FIND_MOTOR,
+    ERROR_INVALID_MAC_CODE,
+    ERROR_NO_BLUETOOTH_ADAPTER,
+    ERROR_NO_DEVICES_FOUND,
+    MotionBlindType,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema({vol.Required(CONF_MAC_CODE): str})
+
+
+class FlowHandler(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for MotionBlinds BLE."""
+
+    VERSION = 1
+
+    _discovery_info: BluetoothServiceInfoBleak | BLEDevice | None = None
+    _mac_code: str | None = None
+    _display_name: str | None = None
+    _blind_type: MotionBlindType | None = None
```

Comment:
better to use instance attributes here
```suggestion
    def __init__(self):
        self._discovery_info: BluetoothServiceInfoBleak | BLEDevice | None = None
        self._mac_code: str | None = None
        self._display_name: str | None = None
        self._blind_type: MotionBlindType | None = None
```